shared/gateway/gateway.go

```diff
@@ -1,29 +1,33 @@
-// Package gateway defines a gRPC gateway to serve HTTP-JSON
-// traffic as a proxy and forward it to a beacon node's gRPC service.
```

Comment:
Why did you remove the documentation? If something is out of date, please update it.

Reply:
Will restore.